src/js/components/profile.js

```diff
@@ -41,7 +41,7 @@ export default class Profile extends Component {
             <TextInput
               value={ firstname }
               placeholder="First name"
-              onChangeText={ e => handleChangeName('firstname', e) }
+              onChangeText={ text => handleChangeName('firstname', text) }
```

Comment:
In other parts of the app that have a value and a category, which way round do the variables go?
```js
doSomething(value, category);
```
or
```js
doSomething(category, value);
```
Just want to make sure we're being consistent